### DISCUSSION TITLE
Welcome to netspeedsimplified Discussions!

### DISCUSSION THREAD

**Discussion by prateekmedia** [prateekmedia/netspeedsimplified#17] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by thiago7828** [comment]
Está dando erro quando vai nas configuraçoẽs da extensão, como faço para corrigir isso? Eu uso o ubuntu 24.04.1 lts e o gnome 46.0

![Captura de tela de 2024-09-29 21-57-10](https://github.com/user-attachments/assets/01ed7aa0-40ea-4cad-88b8-e9a77753077d)

The settings of extension netspeedsimplified@prateekmedia.extension had an error:
```
TypeError: Object 0xe0646d2fd18 is not a subclass of GObject_Object, it's a Promise

Stack trace:
  _wrapWidget@resource:///org/gnome/Shell/Extensions/js/extensions/prefs.js:52:15
  fillPreferencesWindow@resource:///org/gnome/Shell/Extensions/js/extensions/prefs.js:37:27
  _loadPrefs@resource:///org/gnome/Shell/Extensions/js/extensionPrefsDialog.js:41:18
  async*_init@resource:///org/gnome/Shell/Extensions/js/extensionPrefsDialog.js:26:14
  ExtensionPrefsDialog@resource:///org/gnome/Shell/Extensions/js/extensionPrefsDialog.js:17:4
  OpenExtensionPrefsAsync@resource:///org/gnome/Shell/Extensions/js/extensionsService.js:139:33
  async*LaunchExtensionPrefsAsync@resource:///org/gnome/Shell/Extensions/js/extensionsService.js:126:14
  _handleMethodCall@resource:///org/gnome/gjs/modules/core/overrides/Gio.js:373:35
  _wrapJSObject/<@resource:///org/gnome/gjs/modules/core/overrides/Gio.js:408:34
  _init/GLib.MainLoop.prototype.runAsync/</<@resource:///org/gnome/gjs/modules/core/overrides/GLib.js:266:34
  
Caused by: Error: This JS object wrapper isn't wrapping a GObject. If this is a custom subclass, are you sure you chained up to the parent _init properly?
```
